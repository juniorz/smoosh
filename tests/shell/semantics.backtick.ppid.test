set -e

pid1=$($TEST_SHELL -c 'echo $PPID')
echo $pid1 >pid1

$TEST_SHELL -c 'echo $PPID' >pid2

[ $(cat pid1) = $(cat pid2) ]
echo pid1=pid2

(echo $PPID) >ppid
[ $PPID = $(cat ppid) ]
echo ppid=subshell