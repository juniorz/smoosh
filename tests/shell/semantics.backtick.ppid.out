pid1=pid2
ppid=subshell
